packages/dioxus-blitz/Cargo.toml

```diff
@@ -26,6 +26,9 @@ style = { workspace = true }
 tracing = { workspace = true, optional = true }
 blitz = { path = "../blitz" }
 blitz-dom = { path = "../dom" }
+blitz-net = { path = "../net" }
+blitz-traits = { path = "../traits" }
+html-escape = "0.2.13"
```

Comment:
Is `html-escape` actually used in this crate?

Reply:
Nope, forgot to remove it.